src/Razor/test/Microsoft.VisualStudio.LanguageServerClient.Razor.Test/HtmlCSharp/CompletionHandlerTest.cs

```diff
@@ -218,8 +218,8 @@ @using System
 
         // Assert
         var vsCompletionList = Assert.IsType<OptimizedVSCompletionList>(result.Value.Value);
-        var item = vsCompletionList.Items.First();
-        Assert.Equal("DateTime", item.Label);
+        var item = vsCompletionList.Items.Where(item => item.Label == "DateTime");
```

Comment:
I'm not sure why the ordering of C# completion items changed, but it doesn't affect the end result in practice since the client seems to sort the items afterwards

Reply:
Isn't the problem that we're getting an unfiltered completion list instead of one filtered to DateTime?

Reply:
Oh, it is ordering things differently. I bet it's just a change in Roslyn.